app/src/main/res/menu/item_menu.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/item_menu">
+
+    <item
+        android:id="@+id/item_menu_view"
```

Comment:
not sure if we should only have 2 items (view or cancel)
In the archived view we should also have the option to delete

Reply:
This can easily be changed in future sprints